docs/azure/sdk/authentication/create-token-credentials-from-configuration.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Create token credentials from configuration
+description: This article describes how to create token credentials from configuration files
+ms.topic: how-to
+ms.custom: devx-track-dotnet, engagement-fy23
+ms.date: 1/16/2024
+---
+
+# Create Azure credential types using configuration files
+
+The `Microsoft.Extensions.Azure` library supports creating different <xref:Azure.Core.TokenCredential?displayProperty=fullName> types from values defined in _appsettings.json_ and other configuration files. This article describes the support for different `TokenCredential` types and how to configure the required values for each type.
+
+## Support for Azure credentials through configuration
+
+The `Microsoft.Extensions.Azure` library can automatically provide Azure service clients with a `TokenCredential` class by searching _appsettings.json_ or other configuration files for credential values using the default `IConfiguration` service for .NET. This approach allows developers to explicitly set credential values across different environments through configuration rather than through app code directly.
+
+The following credential types are supported via configuration:
+
+* <xref:Azure.Identity.ManagedIdentityCredential?displayProperty=fullName>
```

Comment:
I recommend alphabetizing this list and linking to the subsections in this doc instead of to the API ref pages